pkgs/misc/vscode-extensions/nix-prefetch-vscode-mktplc/nix-prefetch-vscode-mktplc

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

/bin/sh is bash on NixOS and something else on most other distros.

Reply:
Applied.